CDTDatastore/query/CDTQProjectedDocumentRevision.h

```diff
@@ -28,12 +30,14 @@
 /**
  Initialise with a datastore so mutableCopy can return a full document.
  */
-- (id)initWithDocId:(NSString *)docId
-         revisionId:(NSString *)revId
-               body:(NSDictionary *)body
-            deleted:(BOOL)deleted
-        attachments:(NSDictionary *)attachments
-           sequence:(SequenceNumber)sequence
-          datastore:(CDTDatastore *)datastore;
+- (nullable instancetype)initWithDocId:(NSString *)docId
```

Comment:
perhaps however when I looked at the header, it doesn't have nullability annotations or an audited region, implying that it can return null, although that might suggest that it's only to allow you to return null